cmd/kops/export_kubecfg.go

```diff
@@ -116,7 +120,12 @@ func RunExportKubecfg(ctx context.Context, f *util.Factory, out io.Writer, optio
 			return err
 		}
 
-		conf, err := kubeconfig.BuildKubecfg(cluster, keyStore, secretStore, &commands.CloudDiscoveryStatusStore{}, buildPathOptions(options))
+		conf, err := kubeconfig.BuildKubecfg(cluster, keyStore, secretStore, &commands.CloudDiscoveryStatusStore{}, buildPathOptions(options), options.admin)
+		if options.admin {
```

Comment:
Given that `--admin` and `--user` are mutually exclusive, we should probably return an error if both are specified.

Reply:
Yep. Done.